src/django/api/tests/test_facility_history_endpoint.py

```diff
@@ -1136,3 +1137,13 @@ def test_in_group_receives_200(self):
             history_response.status_code,
             200,
         )
+
+
+class DescribeChangeValueTests(FacilityAPITestCaseBase):
```

Comment:
Each class should have a separate file or you can make these tests as part of the  FacilityHistoryEndpointTest class.

Reply:
Removed separate class, added tests to Facility history endpoint test